commands/license_info

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+help() {
+  cat <<EOF
+
+----------------------------------------------------------------------------------------------------------
+HELP MESSAGE:
+
+    Usage: c8y oee $(basename "$0") [--view <json|table>]
+
+    Arguments:
```

Comment:
- do you think it would be possible to add the `--select` argument, e.g. `c8y oee-cli.naph license_info --select licenseStatus` to allow the user to specify which values should be shown?